### DISCUSSION TITLE
Dummy (Neutral) & Robot (Neutral)

### DISCUSSION THREAD

**Discussion by Neon-Catt** [Alf21/TTT2#951] [Role Ideas]
**Description** It was designed as a machine to learn how to survive everything you throw at it, but it turned against it's owners and now is seeking to destroy terrorist town. But first it has to learn how...

**Win condition and alignment** The dummy plays on its own team and cannot win without changing teams. The robot plays on its own team and wins after all other teams are dead.

**Abilities and gameplay** 

- The dummy starts with 50 health and can respawn 5+ times.

- On it's last respawn, it turns into the robot with 300 health and 2x damage, and all players are alerted that it has spawned.

- The robot has an infinite ammo, high fire-rate, moderate damage chaingun which cannot be dropped.

- The dummy has a custom shop, and keeps credits after dying even if another role with a shop identifies it's body.

- It spawns with an item that allows it to disguise as other players, that cannot be dropped and does not drop on it's death.

- Any non-player caused death respawns the dummy, but does not bring it closer to receiving the robot role.

- Similar to the jester, the dummy can fire guns and other weapons, but does no damage to other players with them.

- The dummy receives 1 credit (Can be changed with convars) after being killed by a player. It's shop items will help it get killed more times.

**Comment by Neon-Catt** [comment]
If this role is created, there should be a convar to change between the robot spawning, and the dummy just getting an instant win. I had the idea for the robot to make things more interesting than an instant win for the dummy.

**Reply by PROFI5** [reply]
i know thats not so important but could it been done if there is some cases where everyone dies at the same time the dummy can get his own win? Like a nova exploding the T and himself while only dummy is alive
i would find it cool if it got 1 way to win as a dummy and a way to  win as a robot

**Reply by Neon-Catt** [reply]
possibly, maybe make it so for 15-30 seconds after all other teams except the dummy and 1 other are dead, the round still goes on and the dummy can try to kill the last team or be killed by them and turn into robot
